api/v1beta1/flowcollector_types.go

```diff
@@ -233,6 +234,25 @@ type FlowCollectorKafka struct {
 	TLS ClientTLS `json:"tls"`
 }
 
+type FlowCollectorIPFIXReceiver struct {
+	//+kubebuilder:default:=""
+	// address of the ipfix external receiver
+	TargetHost string `json:"targetHost"`
+
+	// port for the ipfix external receiver
+	TargetPort int `json:"targetPort"`
+
+	// Transport protocol (tcp/udp) to be used for the IPFIX connection, defaults to tcp
+	// +unionDiscriminator
+	// +kubebuilder:validation:Enum:="TCP";"UDP"
+	// +optional
+	Transport string `json:"transport,omitempty"`
+
+	// EnterpriseId,omitempty" doc:"Enterprise ID for exporting transformations
+	//+optional
+	EnterpriseID int `json:"enterpriseId,omitempty"`
```

Comment:
As mentioned here https://github.com/netobserv/network-observability-operator/issues/296#issuecomment-1498920122 ; the enterpriseID might perhaps have to be hardcoded instead

Reply:
@jotak , sorry I didn't see that comment. I'm working on it.

Reply:
@jotak @jpinsonneau I added the hard coded enterprise ID with 27027 as mentioned in the [#296 (comment)](https://github.com/netobserv/network-observability-operator/issues/296#issuecomment-1498920122) but looking into the [RFC5153](https://datatracker.ietf.org/doc/html/rfc5153#section-9.2) and the IANA for [27027](https://www.iana.org/assignments/enterprise-numbers/?q=27027)  I get this organization `ESTEAM Software GmbH`. Not sure if that is the intended effect and even if this should actually be hardcoded. Let me know if that makes sense. And don't hesitate to ask for any other changes.

Reply:
@jpinsonneau @acmenezes - Thanks folks - We're good to use 2 as the enterprise ID. I've got this registered with the IBM numbers person 👍

Reply:
@jpinsonneau @rupertgregoryibm , let me see if I understood this well. We're letting 2 as the default value and let it configurable right? As I also understand other companies may configure with their registered number. Does that make sense?

Reply:
@acmenezes as we have the validation, we can simply force it to `2`. No need to configure it :smile: